src/devices/Arduino/README.md

```diff
@@ -0,0 +1,106 @@
+# SPI, GPIO and I2C drivers for Arduino
+
+This binding supports GPIO, PWM, SPI and I2C access from a normal Desktop environment (Windows, Linux) trough an Arduino board.
+
+## Device family
+
+This binding remotely controls different Arduino boards directly from PC Software. It provides support for accessing GPIO ports as well as I2C devices, SPI devices, PWM output and analog input. The Arduino is remote controlled by individual commands from the PC, the entire program will run on the PC, and not on the Arduino, so the connection cannot be removed while the device is being used. 
+
+## Desktop Requirements
+
+In order to have an Arduino board working with the PC, you need to Install the Arduino IDE together with the drivers for your board type. If you get a simple sketch uploaded and running (such as the blinking LED example) you are fine to start. If you're new to the Arduino world, read the introduction at https://www.arduino.cc/en/Guide for a quick start. The explanations below assume you have the Arduino board connected trough an USB cable with your PC and you know how to upload a sketch. 
```

Comment:
technically this is not needed. Once the Arduino is flashed you don't need Arduino IDE

Reply:
True, but you might still need to install the USB driver. I have never tried installing that without the IDE.

Reply:
Added a sentence saying that this is (usually) a one-time task.

Reply:
looks good now